test/test_helper.rb

```diff
@@ -11,8 +15,11 @@
 ActionMailer::Base.delivery_method = :test
 I18n.enforce_available_locales = false
 
-$:.unshift File.dirname(__FILE__) + '/../lib'
-require 'mail_form'
+module Rails
+  def self.env
+    @env ||= ActiveSupport::StringInquirer.new('test')
+  end
+end
```

Comment:
Yes, we have a `Rails.env.development?` check - what changed with 4.2 is that now the `Rails` module is defined by the html sanitization gem, and we get a `NoMethodError` in the tests. We could throw a `respond_to?` check in the gem code but I rather leave this setup mismatch fix in the test suite itself.